bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -478,6 +479,7 @@ public synchronized BigtableTableAdminClient getTableAdminClient() throws IOExce
    * @return a {@link BigtableTableAdminClientWrapper} object.
    * @throws java.io.IOException if any.
    */
+  @InternalApi("For internal usage only")
   public synchronized IBigtableTableAdminClient getTableAdminClientWrapper() throws IOException {
```

Comment:
The interface IBigtableTableAdminClient should also be InternalOnly

Reply:
Thanks for the feedback, I have marked `IBigtableTableAdminClient` and `IBigtableDataClient` as `@InternalApi`. Please have a look.